srcpkgs/e2tools/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'e2tools'
+pkgname=e2tools
+version=0.0.16
+revision=1
+build_style=gnu-configure
+makedepends="e2fsprogs-devel"
+short_desc="Utilities for manipulating files in an ext2/ext3 filesystem"
+maintainer="Daniel Santana <daniel@santana.tech>"
+license="GPL-2.0-only"
+homepage="http://home.earthlink.net/~k_sheff/sw/e2tools/"
+distfiles="http://home.earthlink.net/~k_sheff/sw/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=4e3c8e17786ccc03fc9fb4145724edf332bb50e1b3c91b6f33e0e3a54861949b
+
+post_install() {
+	for man in e2cp.1 e2ln.1 e2ls.1 e2mkdir.1 e2mv.1 e2rm.1 e2tail.1 e2tools.7; do
```

Comment:
Is it really necessary to vendor the manpages ? they might be obsolete by the next version and we might not notice.

Reply:
I don't know if there will be an next version of e2tools as the last commit was around 2004, and the manpages are a contribution from Debian. I couldn't find any alternative source with the manpages, as Debian adds it with a patch and Arch Linux (on AUR) also vendors them.

Reply:
sounds fine by me then